### ISSUE TITLE
[ BUG ] `New-FalconScheduledScan` not sending `endpoint_notification` boolean

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#364]
**Describe the bug**
When using `New-FalconScheduledScan`, the value of `endpoint_notification` (via the `Notification` parameter) is not being sent

**Environment (please complete the following information):**
 - OS: Windows 10, Windows 11
 - PowerShell: 5.1
 - PSFalcon: 2.2.5

**Comment by bk-cs** [comment]
This is fixed in the v2.2.6 release. To fix this issue in your local module modify the `Public\ods.ps1` file under the `New-FalconScheduledScan` function and change...
```powershell
Format = @{
  Body = @{
    root = @('file_paths','host_groups','max_file_size','max_duration','pause_duration',
      'cloud_ml_level_detection','cloud_ml_level_prevention','sensor_ml_level_detection',
      'sensor_ml_level_prevention','cpu_priority','scan_exclusions','description',
      'schedule','ignored_by_channel_file')
  }
}
```
To include `endpoint_notification`:
```powershell
Format = @{
  Body = @{
    root = @('file_paths','host_groups','max_file_size','max_duration','pause_duration',
      'cloud_ml_level_detection','cloud_ml_level_prevention','sensor_ml_level_detection',
      'sensor_ml_level_prevention','cpu_priority','scan_exclusions','description',
      'schedule','ignored_by_channel_file','endpoint_notification')
  }
}
```

**Comment by bk-cs** [comment]
Resolved in 2.2.6 release which has been published to GitHub and will be published on the PowerShell Gallery soon.
